prime-angular-frontend/src/app/shared/components/forms/email-form/email-form.component.html

```diff
@@ -0,0 +1,19 @@
+<ng-container [formGroup]="form">
+  <app-form-icon-group (event)="removeEmailInput()"
+                       [show]="showRemoveButton">
+    <div class="row"
+         testing>
+      <div class="col">
+
+        <mat-form-field class="w-100">
+          <input matInput
+                 placeholder="Email"
+                 formControlName="email"
+                 maxlength="50">
+          <mat-error *ngIf="email.errors?.['required']">Required</mat-error>
```

Comment:
The convention used in the project seems to be `email.hasError('required')`

Reply:
code is updated.